src/backend/people/settings.py

```diff
@@ -280,6 +281,7 @@ class Base(Configuration):
     EMAIL_FROM = values.Value("from@example.com")
 
     AUTH_USER_MODEL = "core.User"
+    INVITATION_VALIDITY_DURATION = timedelta(days=7)
```

Comment:
I added it as timedelta to be able to import it directy whenever needed. Still very readable and easy to modify.

Reply:
I think it should be a duration in seconds so it can be set via environment variable. 
The timedelta can be instantiated on the use site with something like:
```
instattion_validity_duration = timedelta(seconds=settings.INVITATION_VALIDITY_DURATION)
```

Reply:
Done !